db/migrate/20170718142148_create_katello_subscription_facet_pools.rb

```diff
@@ -0,0 +1,15 @@
+class CreateKatelloSubscriptionFacetPools < ActiveRecord::Migration
+  def change
```

Comment:
There shouldn't be, all of this data exists in candlepin and now is going to be mirrored in our database